tests/test_other.py

```diff
@@ -10136,6 +10139,18 @@ def test_shared_flag(self):
     self.assertContained('warning: -shared/-r used with executable output suffix', err)
     self.run_js('out.js')
 
+  def test_shared_and_shared_module_flag(self):
```

Comment:
I think this should be `test_shared_and_side_module_flag` or maybe just `test_shared_and_shared_module`?

Reply:
oh, thanks for pointing this out